lib/blacklight/models/content.rb

```diff
@@ -9,7 +9,9 @@ def initialize(xml)
     end
 
     def canvas_conversion
-      "<a href='$IMS_CC_FILEBASE$/#{@linkname}'>#{@linkname}</a>"
+      # rubocop:disable LineLength
```

Comment:
I was working with ben on this. It could go either way but personally I would just break the rule this time. basically you're breaking it into a bunch of silly vars just to please the linter. maybe the real issue it that a model is returning html. I say that knowing nothing about this repo :)